src/main/java/com/demo/importer/config/ConfigPropertiesInitializer.java

```diff
@@ -0,0 +1,72 @@
+package com.demo.importer.config;
```

Comment:
Names of variables and methods are intuitive 👌 
Code is well organized

Reply:
Thanks you!